File: AI-models/test_recommendation.py
from generate_recommendation import generate_recommendation

result = generate_recommendation(
    business_type="retail",
    category_scores={
        "infrastructure": 60.0,
        "process": 70.0,
        "people": 50.0,
        "data": 65.0,
        "security": 55.0,
    },
    overall_score=60.5,
    maturity_level="Level 4 - Managed",
    gaps=["people", "security"],
)

print("Source:   ", result["source"])
print("Summary:  ", result["summary"])
print("Priorities:")
for p in result["priorities"]:
    print("  -", p)

File: Backend/test-recommendation.js
const { generateRecommendation } = require("./src/services/recommendationService");

async function test() {
    const result = await generateRecommendation(
        "Micro",
        {
            infrastructure: 60,
            process: 70,
            people: 50,
            data: 65,
            security: 55
        },
        60.5,
        "Level 4 - Managed",
        []
    );

    console.log(JSON.stringify(result, null, 2));
}

test();